class Except4 {
  void m() throws Except4 { }
}
